### PR TITLE
✨  Add Wunderkind ads extension

### LINKED ISSUE #34871: wunderkind extension for amp-ad
### Summary

Provides support for `<amp-ad type="wunderkind" />`

[Wunderkind](https://wunderkind.co) provides configurable ads for our clients using an auction system 

### Design Document

_No response_

### Motivation

AMP is a large portion of our clients' traffic and they have been requesting that we support AMP

### Alternative Solutions

We didn't consider any other options to bring our ads to AMP. Our ads can be backed by many different networks and there are not any existing AMP extensions to allow this

### Launch Tracker

_No response_

### Notifications

/cc @ampproject/wg-approvers @ampproject/wg-ads-reviewers

### PR DESCRIPTION
**Pull request by weirdian2k3**
This PR provides support for <amp-ad type="wunderkind" />

Closes #34871

### CODE REVIEW

**Comment by weirdian2k3** [comment]
hi @lannka , do you think you'll be able to take a look at this this week? I have a follow-up PR to file once this is merged

File: ads/vendors/wunderkind.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/wunderkind.js:2]: nit: 2020

File: ads/vendors/wunderkind.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/wunderkind.md:2]: nit:2020

**Comment by weirdian2k3** [comment]
@calebcordry Hi Caleb! This is the PR I mentioned

**Review by calebcordry — CHANGES_REQUESTED** [review]
Just a couple nits

**Review by calebcordry — APPROVED** [review]

**Comment by weirdian2k3** [comment]
Hi @calebcordry !

Would you be able to see what is causing these failures?

**Comment by calebcordry** [comment]
@weirdian2k3 this appears to be some unrelated babel error. Can you try syncing to latest master? Let me know if you have issues, thanks.

**Comment by weirdian2k3** [comment]
@calebcordry updated to master, thank you!

**Comment by weirdian2k3** [comment]
@calebcordry if you get a moment, could you approve the workflow so that I can see if the update fixed it?

File: 3p/vendors/wunderkind.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// src/polyfills.js must be the first import.
+import '../polyfills';
+
+import {draw3p, init} from '../integration-lib';
```
Comment (calebcordry) [inline 3p/vendors/wunderkind.js:20]: ah I think these were updated while you were working on this PR. Looks like they are now `#3p/integration-lib`, `#3p/3p`, and `#ads/vendors/wunderkind`

**Comment by weirdian2k3** [comment]
@calebcordry thank you for catching that! I've updated the file, which disabled auto-merge

**Comment by calebcordry** [comment]
looks like now we have some formatting errors :( you can expose them by running `amp prettify` or following the ciricleci link above.

**Comment by weirdian2k3** [comment]
@calebcordry 14th times the charm, I've updated the formatting

**Comment by calebcordry** [comment]
🎉 Thanks for contributing!

**Pull request merged**